org.exaile.Exaile.json

```diff
@@ -0,0 +1,112 @@
+{
+  "app-id": "org.exaile.Exaile",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "exaile",
+  "rename-desktop-file": "exaile.desktop",
+  "rename-appdata-file": "exaile.appdata.xml",
+  "rename-icon": "exaile",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--system-talk-name=org.freedesktop.UDisks2",
+    "--filesystem=xdg-music"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig"
+  ],
+  "modules": [
+    {
+      "name": "BerkeleyDB",
+      "subdir": "dist",
+      "builddir": true,
+      "config-opts": [
+        "--enable-compat185",
+        "--enable-shared",
+        "--enable-cxx",
+        "--enable-dbm",
+        "--enable-stl"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.oracle.com/berkeley-db/db-5.3.28.tar.gz",
+          "sha256": "e0a992d740709892e81f9d93f06daf305cf73fb81b545afe72478043172c3628"
+        },
+        {
+          "type": "patch",
+          "path": "atomic.patch"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "cp -p /usr/share/automake-*/config.{sub,guess} dist"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "python3-bsddb3",
+      "buildsystem": "simple",
+      "build-options": {
+        "no-debuginfo": true,
+        "env": {
+          "BERKELEYDB_DIR": "/app"
+        }
+      },
+      "build-commands": [
+        "pip3 install --no-deps --no-use-pep517 --prefix=/app ."
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://files.pythonhosted.org/packages/source/b/bsddb3/bsddb3-6.2.7.tar.gz",
+          "sha256": "b0f7fa63eb240cd5815809c9d1d7df9f7cc8d6fa9619d0edbe9c794afc02dc9f"
+        }
+      ]
+    },
+    "python3-pycairo.json",
+    "python3-gobject.json",
+    "python3-PyGObject.json",
+    "python3-mutagen.json",
+    "python3-dbus-python.json",
+    "python3-feedparser.json",
+    "python3-ipython.json",
+    "python3-pylast.json",
+    "python3-lxml.json",
+    "python3-beautifulsoup4.json",
+    "python3-musicbrainzngs.json",
```

Comment:
Any reason not to merge this into a single file? (Also, are ipython and poetry actually required on runtime?)

Reply:
This is the default output of https://github.com/flatpak/flatpak-builder-tools/tree/master/pip. ipython is (optionally) required for the ipconsole plugin. Removed poetry.